game-app/game-headed/src/test/java/games/strategy/triplea/HeadedResourceLoaderTest.java

```diff
@@ -0,0 +1,12 @@
+package games.strategy.triplea;
+
+import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
+
+/** Unit tests for the {@link ResourceLoader} class when running the game Headed. */
+final class HeadedResourceLoaderTest {
+  @Test
+  void canLoadImage() {
+    ResourceLoader.loadImageAsset(Path.of("launch_screens", "triplea-logo.png"));
```

Comment:
Shouldn't the strings be constants somewhere or how do we know in case they are renamed this test should be adjusted?